Sources/NIO/DatagramVectorReadManager.swift

```diff
@@ -78,25 +86,40 @@ struct DatagramVectorReadManager {
     /// - parameters:
     ///     - socket: The underlying socket from which to read.
     ///     - buffer: The single large buffer into which reads will be written.
-    func readFromSocket(socket: Socket, buffer: inout ByteBuffer) throws -> ReadResult {
+    ///     - reportExplicitCongestionNotifications: Should explicit congestion notifications be reported up using metadata.
+    func readFromSocket(socket: Socket,
+                        buffer: inout ByteBuffer,
+                        reportExplicitCongestionNotifications: Bool) throws -> ReadResult {
         assert(buffer.readerIndex == 0, "Buffer was not cleared between calls to readFromSocket!")
 
         let messageSize = buffer.capacity / self.messageCount
+        let controlBytesPerMessage =
+            (self.controlMessageBuffers.count / self.messageCount / MemoryLayout<cmsghdr>.alignment)
+            * MemoryLayout<cmsghdr>.alignment
```

Comment:
Why does this divide out and then remultiply by the alignment of the cmsghdr? What's the goal, to pad out to that alignment value? Whatever the reason, let's throw a comment in here explaining why we did it.